package.json

```diff
@@ -22,16 +22,13 @@
   },
   "homepage": "https://github.com/ember-cli/ember-cli-preprocessor-registry#readme",
   "dependencies": {
-    "broccoli-clean-css": "^1.1.0",
-    "broccoli-funnel": "^1.0.0",
-    "broccoli-merge-trees": "^1.0.0",
-    "debug": "^2.2.0",
-    "ember-cli-lodash-subset": "^1.0.7",
-    "process-relative-require": "^1.0.0",
-    "silent-error": "^1.0.0"
+    "broccoli-clean-css": "^2.0.1",
```

Comment:
it turns out that this was even more of a breaking change than I thought (https://github.com/jakubpawlowicz/clean-css#important-40-breaking-changes). we are exposing the `clean-css` options via `minifyCSS.options` and since the options have changed this in itself was a breaking change.